packages/babel-preset-react-app/create.js

```diff
@@ -29,6 +29,7 @@ module.exports = function(api, opts, env) {
   var isEnvProduction = env === 'production';
   var isEnvTest = env === 'test';
 
+  var allowESModules = validateBoolOption('allowESModules', opts.allowESModules, true);
```

Comment:
Let's make this default to `isEnvDevelopment || isEnvProduction` and use the existing name.
```suggestion
  var useESModules = validateBoolOption('useESModules', opts.useESModules, isEnvDevelopment || isEnvProduction);
```